docs/developers/CONTRIBUTING.md

```diff
@@ -45,25 +45,28 @@ Code will also be linted to enforce the stylistic and logistical rules specified
 
 If you wish to tell the linter to ignore a specific line use the `# noqa` comment along with the specific error code (e.g. `import sys  # noqa: E402`) but please do not ignore errors lightly.
 
-## Building the icons
+## Adding icons
 
-Our icon build process starts with a base set of `svg` formatted icons stored in `napari/resources/icons/svg`. If you want to add a new icon to the app, make the icon in whatever program you like and add it to that folder.
+If you want to add a new icon to the app, make the icon in whatever program you
+like and add it to `napari/resources/icons/`.
```

Comment:
Do icons still need to be svg files?

Reply:
If so probably worth adding a note about that here

Reply:
yes, must be svg. will leave a note.